components/ToastConfig.tsx

```diff
@@ -1,12 +1,25 @@
-import { ToastConfig } from "react-native-toast-message";
-import { Text } from "./ui/text";
+import { Link } from "expo-router";
 import { View } from "react-native";
+import {
+  BaseToast,
+  BaseToastProps,
+  ToastConfig,
+} from "react-native-toast-message";
+import { useAppStore } from "~/lib/state/appStore";
+
 import { CheckCircle, XCircle } from "./Icons";
-import { Link } from "expo-router";
 import { Button } from "./ui/button";
-import { useAppStore } from "~/lib/state/appStore";
+import { Text } from "./ui/text";
 
 export const toastConfig: ToastConfig = {
+  info: (props: BaseToastProps) => (
+    <BaseToast
+      {...props}
+      text2NumberOfLines={2}
+      text1Style={{ fontSize: 16 }}
+      text2Style={{ fontSize: 14 }}
+    />
+  ),
```

Comment:
I found the default text size for the info toast was unreadable, so I made it a little bit bigger.